### DISCUSSION TITLE
POLL - What platform are you deploying the chart on?

### DISCUSSION THREAD

**Discussion by thesuperzapper** [airflow-helm/charts#565] [Polls]
This poll helps us understand what platforms are most popular for the `User-Community Airflow Helm Chart`.
This helps us prioritize efforts and influences what features we add next!

**Comment by karakanb** [comment]
I deploy on DigitalOcean Kubernetes.

**Comment by emartgu** [comment]
RKE cluster on-prem.

**Comment by BELAGGOUN** [comment]
I'm deploying on Azure Redhat Openshift Cluster.

**Comment by mavenzer** [comment]
Openshift Cluster

**Reply by BELAGGOUN** [reply]
Hi @mavenzer  , I'm struggling with Airflow Helm chart deployment on Openshift , I have issues with the UID/GID and fsGroup being blocked by the Openshift scc . were you able to do it successfully ? can you please share with me some tips on that ? Thank you

**Reply by mavenzer** [reply]
Hi @BELAGGOUN, So the chart we have deployed is a custom airflow chart that Airflow and Bitnami repo heavily inspire. I agree there is an issue with fsGroup in Openshift. You probably need to know the range for the uid/gids accessible for your namespace.  To know the uids/gids which are accessible. you can `oc describe project your_namespace`. This will probably list out the range for the uids and gids. If you have admin rights to the cluster, you can add your own PSP. My recommendation would be to try to understand the airflow charts and if possible also go with bitnami repo, which is easier to understand IMO.

**Reply by yogeshwariingale** [reply]
@BELAGGOUN You can add adm policy to serviceAccount. It solves issue. I had same issue it got resolved with below command
`oc adm policy add-scc-to-user anyuid -z serviceAccountname_usedinairflow -n your_airflow_namespace`

hope it helps

**Comment by agrounds** [comment]
My org deploys to EKS, but also to Docker Desktop's single node Kubernetes cluster for local development
